doc/source/tune/api_docs/logging.rst

```diff
@@ -3,143 +3,30 @@
 Loggers (tune.logger)
 =====================
 
-Tune has default loggers for TensorBoard, CSV, and JSON formats. By default, Tune only logs the returned result dictionaries from the training function.
+Tune has default loggers for TensorBoard, CSV, and JSON formats.
```

Comment:
Maybe "Tune automatically uses loggers for"? "has" suggests that only those are implemented in the library